### ISSUE TITLE
About the TCN part

### ISSUE THREAD

**Issue by JosephKKim** [3dpose/GnTCN#13]
Thanks for making your amazing work open in public! 
I am a newcomer to this topic(human pose estimation) so I'm not familiar with reading the code.
And I have a question about the TCNs part.
Even after I've read your paper thoroughly, I am not quite sure about the TCN part of this model architecture.

On page 5 of your paper, there is equation (8) which handles the importance of the output from Root-TCN and velocity-TCN.
I wonder where I can find this part in the code.

Thanks!

**Comment by 3dpose** [comment]
Thank you for your interest in our work. 

In the released code, we did not include the velocity TCN part as we found that the velocity TCN does not affect the final result as much as other modules after we re-train the 2D pose estimator (HRNet). In particular, the evaluation results reported in our paper was based on the 2D estimator trained only on MPII dataset. In the released code/models, the 2D estimator is trained on MPII and MuCo-3DHP, where MuCo-3DHP is the training dataset of MuPoTs-3D (i.e., MuPoTs-3D is the corresponding test set of MuCo-3DHP). 

With the improved 2D estimator, we're able to achieve the same performance as reported in the paper without the velocity TCN part. Thus, to keep the repository simple, we only include the GCN and the Root-TCN. Hope this clarifies your question.

**Comment by JosephKKim** [comment]
Thank you so much for the explanation. 
Now, I can see everything is clear. 

The reason why I asked you the question is that I was interested in the occlusion time (start, end) that your group defined and giving weights to the estimated pose depending on the distance from the boundary. 
So I was wondering how it was implemented in the code. 

Thanks again for the reply!

I'll close the issue.
